### PR TITLE
[[ Bug 19704 ]] Fixed HiDPI Win32 bug

### PR DESCRIPTION
**Pull request by abrisan**

### CODE REVIEW

File: engine/src/w32stack.cpp [hunk 1]
```diff
@@ -603,8 +603,12 @@ void MCStack::constrain(intptr_t lp)
 
 	// MW-2007-07-27: In Windows 98 we need to clamp to 32767...
 	
-	mmptr -> ptMinTrackSize . x = minwidth + dx;
-	mmptr -> ptMinTrackSize . y = minheight + dy;
+	double *t_screenPixelScale;
+	uindex_t t_count;
+	MCResListScreenPixelScales(false, t_count, t_screenPixelScale);
+
+	mmptr -> ptMinTrackSize . x = minwidth * (*t_screenPixelScale) + dx;
```
Comment (montegoulding) [inline engine/src/w32stack.cpp]: @livecodealex we get the correct display above on L580 so we don't need to do this. Just `&t_display -> pixel_scale` should cover us.

**Comment by montegoulding** [comment]
@livecode-vulcan review ok e18820b

**Comment by livecode-vulcan** [comment]
💙 review by @montegoulding ok e18820b59ed38c2cd00a6f3aad9d04236c6b2eff

**Comment by livecode-vulcan** [comment]
😎 test success e18820b59ed38c2cd00a6f3aad9d04236c6b2eff

* try-community-armv6-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-sdk26_ndk16r15/builds/167)
* try-community-armv7-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv7-android-sdk26_ndk16r15/builds/159)
* try-community-arm64-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-arm64-android-sdk26_ndk16r15/builds/159)
* try-community-x86-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86-android-sdk26_ndk16r15/builds/159)
* try-community-x86_64-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86_64-android-sdk26_ndk16r15/builds/159)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/1681)
* try-community-universal-ios-iphoneos12.0: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos12.0/builds/63)
* try-community-universal-ios-iphonesimulator12.0: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator12.0/builds/63)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/1711)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/1704)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/1701)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/1737)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/1678)

**Pull request merged**